Examples/HackerNewsClient/HackerNewsClient/AppDelegate.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  AppDelegate.swift
+//  HackerNewsClient
+//
+//  Created by Francesco Di Lorenzo on 14/11/2016.
+//  Copyright © 2016 BendingSpoons. All rights reserved.
+//
+
+import UIKit
+import Katana
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+    var renderer: Renderer?
+    
+    func application(_ application: UIApplication,
+                     didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        
+        self.window = UIWindow(frame: UIScreen.main.bounds)
+        self.window?.rootViewController = UIViewController()
+        self.window?.rootViewController?.view.backgroundColor = UIColor.white
+        self.window?.makeKeyAndVisible()
+        
+        let view = (self.window?.rootViewController?.view)!
+        
+        let store = Store<HackerNewsState>(middlewares: [], dependencies: PostsProvider.self)
+        
+        self.renderer = Renderer(rootDescription: HackerNews(props: HackerNews.Props.build({
```

Comment:
I'd split the description initialisation from the renderer one. It is easier to read